lines.js

```diff
@@ -6,6 +6,7 @@ var colour;
 var colourPanel = document.querySelector("#colours-panel");
 var sizePanel = document.querySelector("#dot-size-panel");
 var dotSize;
+var radius = document.querySelector(".dot-size.chosen circle").getAttribute("r");
```

Comment:
I would use `data-` attribute here. Getting attribute of SVG sub-element is not good approach